docs/permissions/permission_use_cases.md

```diff
@@ -155,6 +155,16 @@ that only allows Content items of type **Image**.
 This ensures that only files of type `image` can be uploaded,
 and only to the **Pictures** Folder.
 
+## Taxonomies
+
+You can control what users or user groups can work with taxonomies. If you want users to be able to create and assign tags, use the following permissions:
+
+- `assign` - to allow user to tag and utag content
+- `read` -  to see the Taxonomy interface
+- `manage` - to create, edit and delete tags
```

Comment:
I'd recommend using phrase "taxonomy entry" instead of "tag" to avoid confusion with Tags which are just an example of a taxonomy.